codec-socks/src/main/java/io/netty/handler/codec/socks/SocksCommonUtils.java

```diff
@@ -62,4 +65,14 @@ static String readUsAscii(ByteBuf buffer, int length) {
         buffer.skipBytes(length);
         return s;
     }
+
+    public static int readUnsignedShortBE(ByteBuf buf) {
```

Comment:
I wonder if we should move these to `ByteBufUtil` to keep things consistent ?

Reply:
Also please add javadocs to public methods

Reply:
> I wonder if we should move these to `ByteBufUtil` to keep things consistent ?

I can definitively do that. I wasn't sure if adding new `*BE` methods in the generic `ByteBufUtil` was still accepted.

> Also please add javadocs to public methods

Will address